web/src/app/forms/FormContainer.js

```diff
@@ -10,12 +10,15 @@ import { get, set } from 'lodash-es'
 export class FormContainer extends React.PureComponent {
   static propTypes = {
     value: p.object,
-    errors: p.arrayOf(
+
+    errors: p.oneOfType([
       p.shape({
-        field: p.string.isRequired,
         message: p.string.isRequired,
+        field: p.string.isRequired,
       }),
-    ),
+      p.node,
+    ]),
```

Comment:
Instead of overriding all of the error processing with a React node, maybe we can enhance it to support a link.

We should continue to display the message provided by the backend and attach it to the proper field, but we can add the link to the end.

> contact method already exists for that type and value [Admin McAdminFace](https://example.org)

```
p.shape({
  message: p.string.isRequired,
  field: p.string.isRequired,
  helpLink: p.shape({text: p.string.isRequired, to: p.string.isRequired})
})
```

Reply:
On top of this, it would be helpful to append the link icon at the end of the field error so indicate it's different (and action can be taken on the text) to the user